src/query/server/server.go

```diff
@@ -140,6 +140,14 @@ func Run(runOpts RunOptions) {
 		}
 	}()
 
+	iopts := instrumentOptions.SetMetricsScope(
```

Comment:
won't this subscope all the metrics for the whole service with `build_info`?

Reply:
Oh I see its a new var. Yeah either dont subscope or just give the var a name that implies it should not be used for anything other than the build reporter